pkg/operation/botanist/component/nginxingressshoot/nginxingress.go

```diff
@@ -154,16 +154,18 @@ func (n *nginxIngress) computeResourcesData() (map[string][]byte, error) {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: configMapName,
 			Labels: map[string]string{
-				v1beta1constants.LabelApp:             labelAppValue,
-				labelKeyRelease:                       labelValueAddons,
-				labelKeyComponent:                     labelValueController,
-				references.LabelKeyGarbageCollectable: references.LabelValueGarbageCollectable,
+				v1beta1constants.LabelApp: labelAppValue,
+				labelKeyRelease:           labelValueAddons,
+				labelKeyComponent:         labelValueController,
 			},
 			Namespace: metav1.NamespaceSystem,
 		},
 		Data: n.values.ConfigData,
 	}
 
+	// We don't call kubernetesutils.MakeUnique() here because the configmap needs to be mutable, since the nginx controller
+	// mutates it in some cases. See https://github.com/gardener/gardener/pull/7386 for more details.
```

Comment:
Why is it not needed for the `nginx-ingress` in the `garden` namespace in the seed cluster?

Reply:
Because we provide this via config if provider=aws, [ref](https://github.wdf.sap.corp/kubernetes/landscape-setup/blob/846d188f3cea397431fcb1b40da1d3846ab85b4b/components/gardener/gardenlet/generate-values.py#L134-L137).

Reply:
Because we provide this via config if `provider=aws`.

Reply:
Via which config? And how is it done for the shoot's nginx-ingress when it's AWS? What's the difference? 👀